chain/pra/events.go

```diff
@@ -0,0 +1,226 @@
+	ResourceId   types.Bytes32
+	TokenId      types.Bytes
+	Recipient    types.Bytes
+	Metadata     types.Bytes
+	Topics       []types.Hash
+}
+
+type EventGenericTransfer struct {
+	Phase        types.Phase
+	Destination  types.U8
+	DepositNonce types.U64
+	ResourceId   types.Bytes32
+	Metadata     types.Bytes
+	Topics       []types.Hash
+}
+
+type EventRelayerThresholdChanged struct {
+	Phase     types.Phase
+	Threshold types.U32
+	Topics    []types.Hash
+}
+
+type EventChainWhitelisted struct {
+	Phase   types.Phase
+	ChainId types.U8
+	Topics  []types.Hash
+}
+
+type EventRelayerAdded struct {
+	Phase   types.Phase
+	Relayer types.AccountID
+	Topics  []types.Hash
+}
+
+type EventRelayerRemoved struct {
+	Phase   types.Phase
+	Relayer types.AccountID
+	Topics  []types.Hash
+}
+
+type EventVoteFor struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Voter        types.AccountID
+	Topics       []types.Hash
+}
+
+type EventVoteAgainst struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Voter        types.AccountID
+	Topics       []types.Hash
+}
+
+type EventProposalApproved struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Topics       []types.Hash
+}
+
+type EventProposalRejected struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Topics       []types.Hash
+}
+
+type EventProposalSucceeded struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Topics       []types.Hash
+}
+
+type EventProposalFailed struct {
+	Phase        types.Phase
+	SourceId     types.U8
+	DepositNonce types.U64
+	Topics       []types.Hash
+}
+
+// Edgeware specs in https://github.com/hicommonwealth/edgeware-node-types/tree/master/src/interfaces
+// Rust implementation of module_event in https://github.com/hicommonwealth/edgeware-node/tree/master/modules
+// Original go events for ChainBridge https://github.com/ChainSafe/ChainBridge/blob/main/shared/substrate/events.go
+
+type SubstateWithFrontierEventRecord struct {
+	types.EventRecords
+	EVM_Log                             []EventEVMLog
+	Council_Proposed                    []types.EventCollectiveProposed       //nolint:stylecheck,golint
+	Council_Voted                       []types.EventCollectiveVoted          //nolint:stylecheck,golint
+	Council_Approved                    []types.EventCollectiveApproved       //nolint:stylecheck,golint
+	Council_Disapproved                 []types.EventCollectiveDisapproved    //nolint:stylecheck,golint
+	Council_Executed                    []types.EventCollectiveExecuted       //nolint:stylecheck,golint
+	Council_MemberExecuted              []types.EventCollectiveMemberExecuted //nolint:stylecheck,golint
+	Council_Closed                      []types.EventCollectiveClosed         //nolint:stylecheck,golint
+	ChainBridge_FungibleTransfer        []EventFungibleTransfer               //nolint:stylecheck,golint
```

Comment:
Please remove all properties related to ChainBridge such as ChainBridge_FungibleTransfer. We don't need them.